minid_client/minid_client_api.py

```diff
@@ -231,6 +250,18 @@ def update_entity(server, name, entity, email, code, globus_auth_token=None):
             r.status_code, r.text))
 
 
+def ark2minid(identifier):
+    if identifier.startswith("ark:/57799/"):
+        identifier = identifier.replace("ark:/57799/", "minid:")
+    return identifier
+
+
+def minid2ark(identifier):
+    if identifier.startswith("minid:"):
+        identifier = identifier.replace("minid:", "ark:/57799/")
+    return identifier
+
+
```

Comment:
It might be good to make `ark:/57799/` a constant like `MINID_ARK_ID` to make it more obvious why the replace needs to happen. I like how they're now functions.

Reply:
Agreed. Fixed in a4eb257.